packages/get-context/src/index.ts

```diff
@@ -55,8 +55,12 @@ export interface ProjectOptions {
   rootDir: string,
 }
 
+interface HookedOptions {
```

Comment:
```suggestion
interface HookOptions {
```

Please rename throughout the code.